java-build/entrypoint.sh

```diff
@@ -50,13 +50,11 @@ else
   packageVersion="1.0.$productBuildNumber"
 fi
 
-if [ ! -z $productBranch ] && [ $productBranch != "master" ]; then
-  packageVersion="$packageVersion-$productBranch"
-fi
-
-echo "deploying version: '$packageVersion'"
+# deploy to Artifactory only on master or if a dev wants to force a dev branch to build
+if ([ ! -z $productBranch ] && [ $productBranch = "master" ]) || [ $deploy = "force" ]; then
```

Comment:
Idea: A developer could make a temporary CI change to their dev branch and add `DEPLOY="force"`. 

@baizhenyu thoughts?

Reply:
I don't have strong opinion on the approach as long as we can make a temp release.

Reply:
Might be helpful to document this somewhere (i.e. in the Readme?)

Reply:
Updated!